data_model.md

```diff
@@ -226,10 +229,37 @@ for more detailed explanation of forecast parameters):
 Additional parameters, such as model configuration parameters, may be
 specified under the “extra parameters” key.
 
-
 Each Forecast has data associated with it. The format of this data is
 found in the Data Format section below
 
+#### Probabilistic forecasts
+{: .anchor}
+
+The Solar Forecast Arbiter supports the specification of a full
+probabilistic forecast in terms of a cumulative distribution function
+(CDF). The metadata for a probabilistic forecast container is the same
+as for a [single-valued forecast](#single-valued-forecasts) with the
+addition of two attributes:
+
+-   *Axis* - The axis on which the CDF is specified. *x* corresponds to
+    fixed variable values. *y* corresponds to fixed quantiles.
+-   *Points* - The fixed variable values or fixed quantiles at which
+    the forecasts will be made.
+
+The CDF may be specified in terms of its variable values (the
+x-axis) or its in terms of its quantiles (the y-axis). The difference
+between the cases is whether the variable values or the quantiles remain
+the same at all forecast times. For a CDF specified in terms of fixed
+quantiles (e.g. `0.0, 0.1, ... 0.9, 1.0`), the user forecasts each
```

Comment:
Will we require endpoints, i.e., (0, 0) and (max, 1)?  I suggest that we do.

Reply:
I'm not sure. I like that it would add more standards and thus let us make more assumptions. But a user that is only concerned with the central interval does not need to know the end points. That seems like a potential situation we'd come across in a trial.